tests/s3/test_copy_object.py

```diff
@@ -97,8 +96,7 @@ def setup(self):
         self.download_path = os.path.join(
             self.test_dir_path, self.object_name2)
         status, self.response1 = self.create_s3cortxcli_acc(
```

Comment:
CLI Code should be removed

Reply:
this creates iam accounts depending on the s3_engine types.